protocol/0062-SPAM-spam_protection.md

```diff
@@ -80,6 +80,22 @@ The minimum allowed withdrawal amount is `spam.protection.minimumWithdrawalQuant
 
 Any withdrawal request for a smaller amount is immediately rejected.
 
+### Referral spam
+
+The [on-chain referral program](./0083-RFPR-on_chain_referral_program.md) adds three transaction types which can be submitted with no cost/risk to the party:
+
+- `CreateReferralSet`
+- `UpdateReferralSet`
+- `ApplyReferralCode`
+
+To avoid spamming of `CreateReferralSet` and `UpdateReferralSet` transactions, a party must meet the staked governance tokens ($VEGA) threshold set by the network parameter `referralProgram.minStakedVegaTokens`. A party who does not meet this requirement should have any transactions of the aforementioned types pre-block rejected.
+
+To avoid spamming of `ApplyReferralCode`, a party must meet the deposited funds threshold set by the network parameter `spam.protection.applyReferral.min.funds`.  All non-VEGA assets count towards this threshold and balances should be scaled appropriately by the assets quantum. A party who does not meet this requirement should have any transactions of the aforementioned type pre-block rejected. This requirement will be checked against snapshots of account balances taken at a frequency determined by the network parameter `spam.protection.balanceSnapshotFrequency`. This network parameter is a duration (e.g. `5s`, `1m5s`).
+
+Further, each referral transactions is limited to `n` submissions per epoch per party where `n` is controlled by the respective network parameter for the transaction (`spam.protection.max.CreateReferralSet`, `spam.protection.max.UpdateReferralSet`, `spam.protection.max.ApplyReferralCode`).
+
+Any party who submits more than `n` transactions of a single referral transaction type in a single epoch will have all future for should have their transactions pre-block rejected.Any party who manages to fit more then `n` of a single referral transaction type into a single block will be banned for 1/48th of an epoch, or un till the end of the current epoch, whichever comes first.
```

Comment:
I'm assuming I know what you're tying to say but I think the first sentence could be rephrased for clarity. 
You're allowed to submit up to `n` transactions for a given transaction type in a given epoch. Once `n` transactions have been included in a block, all further transactions for this type in the same epoch will be pre-rejected and not included in a block.